pkg/query-service/app/http_handler.go

```diff
@@ -652,6 +652,11 @@ func (aH *APIHandler) queryRangeMetricsV2(w http.ResponseWriter, r *http.Request
 				return
 			}
 			var query bytes.Buffer
+
+			// replace reserved variables
+			metricsQueryRangeParams.Variables["$__from_ts"] = metricsQueryRangeParams.Start
+			metricsQueryRangeParams.Variables["$__to_ts"] = metricsQueryRangeParams.End
```

Comment:
We have provided `{{.SIGNOZ_START_TIME}}` and `{{.SIGNOZ_END_TIME}}` for the templating. I think we should be consistent and use the same templating style.

Reply:
I looked into it. SIGNOZ_START_TIME is nano second and puts too onus on user to really convert the formats (e.g /1000 or convert to time.Time). 

I have added following vars to make it simpler now. we should change SIGNOZ_START_TIME to start_ts_nano or something to indicate what it is. 

start_unix_ts, end_unix_ts - unix timestamps start and ends  
start_datetime, end_datetime - start and end dates in normal date time format 

Interpolating these values will aid writing query conditions easily.